docs/en/latest/plugins/traffic-split.md

```diff
@@ -0,0 +1,178 @@
+---
+title: traffic-split
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+This guide describes how to use the traffic-split plugin in APISIX Ingress.
+
+The [traffic-split](https://apisix.apache.org/docs/apisix/plugins/traffic-split) plugin can be used to dynamically route part of traffic to various upstream services.
+
+:::info IMPORTANT
+
+If multiple services are declared in ApisixRoute `http[].backends`, it cannot be used with traffic-split.
```

Comment:
Maybe this document could also be about traffic splitting using `http[].upstreams`?

Reply:
Do you mean external services?
As far as I know, http []. upstreams can only be used as an external service at present.
https://github.com/apache/apisix-ingress-controller/blob/1.6.0/pkg/providers/apisix/translation/apisix_route.go#L385-L386

Reply:
Yes, instructions for using the traffic split plugin on the external service. Hope to add to that.